### PR TITLE
[nix] Cleanup docs for Nix-based static binary

### PR DESCRIPTION
**Pull request by hswong3i**
Formatting by:
```
pandoc -f gfm -t gfm -o README.md README.md
```

Signed-off-by: Wong Hoi Sing Edison <hswong3i@gmail.com>

### CODE REVIEW

File: install.md [hunk 1]
```diff
@@ -1,63 +0,0 @@
-# conmon Installation Instructions
```
Comment (saschagrunert) [inline install.md:1]: I think we had a reason to put the content into a dedicated `install.md`, right @haircommander @TomSweeneyRedHat ?

Comment (hswong3i) [reply install.md:1]: Oh I don't understand if any latent needs so just simply clean it up together (please kindly correct me) ;-)

Comment (haircommander) [reply install.md:1]: :shrug: I don't remember exactly why we went for an install.md, TBH. a reference to it in the README would probably be good in general. I'm fine with either approach

Comment (hswong3i) [reply install.md:1]: Seems no specific reason: https://github.com/containers/conmon/pull/61

File: README.md [hunk 2]
```diff
@@ -4,14 +4,119 @@
...
+To build the binaries by locally install the nix package manager:
+
+``` shell
+nix build -f nix/
+```
+
+### Ansible
+
+An [Ansible Role](https://github.com/alvistack/ansible-role-conmon) is
+also available to automate the installation of above statically linked
```
Comment (TomSweeneyRedHat) [inline README.md:110]:
```suggestion
also available to automate the installation of the above statically linked
```

File: README.md [hunk 3]
```diff
@@ -4,14 +4,119 @@
...
+
+``` shell
+sudo yum install -y \
+  gcc \
+  git \
+  glib2-devel \
+  glibc-devel \
+  make \
+  pkgconfig \
+  runc
```
Comment (TomSweeneyRedHat) [inline README.md:51]: we've listed runc here and a few other places below.  What about crun?  I think Fedora 32+ might not be happy with runc. 

@rhatdan WDYT?

Comment (rhatdan) [reply README.md:51]: Yes crun would be better.

File: README.md [hunk 4]
```diff
@@ -4,14 +4,119 @@
...
+It is possible to build a statically linked binary of conmon by using
+the officially provided
+[nix](https://nixos.org/nixos/packages.html?attr=conmon&channel=nixpkgs-unstable&query=conmon)
+package and the derivation of it [within this repository](nix/). The
+builds are completely reproducible and will create a x86\_64/amd64
+stripped ELF binary for [glibc](https://www.gnu.org/software/libc).
+
+### Nix
+
+To build the binaries by locally install the nix package manager:
```
[the remark this answers is not in the document]
Comment (hswong3i) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (hswong3i) [reply README.md]: Fixed

File: README.md [hunk 5]
```diff
@@ -4,14 +4,119 @@
 
 An OCI container runtime monitor.
 
-Conmon is a monitoring program and communication tool between a container manager (like [podman](https://podman.io/) or [CRI-O](https://cri-o.io/)) and an OCI runtime (like [runc](https://github.com/opencontainers/runc) or [crun](https://github.com/giuseppe/crun)) for a single container.
+Conmon is a monitoring program and communication tool between a
+container manager (like [podman](https://podman.io/) or
```
[the remark this answers is not in the document]
Comment (hswong3i) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (hswong3i) [reply README.md:51]: BTW crun is NOT always available for most OS: https://pkgs.org/download/crun

How about remove `runc`?

Comment (hswong3i) [reply README.md:51]: I checked both Podman (https://github.com/alvistack/podman.io/blob/master/getting-started/installation.md#build-and-run-dependencies) and CRI-O (https://github.com/cri-o/cri-o/blob/master/tutorials/setup.md#build-and-run-dependencies) also mentioning `runc`.

Due to crun is NOT always available for most OS: https://pkgs.org/download/crun, I would like to suggest just keep it as-is ;-S

**Comment by haircommander** [comment]
LGTM, waiting for final nod from @TomSweeneyRedHat and we'll get this in. Thanks @hswong3i

**Comment by rhatdan** [comment]
@haircommander Are the TEST_USERNS tests supposed to pass?

**Comment by haircommander** [comment]
the cri-o tests are known to be broken

**Pull request merged**